core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -217,6 +217,7 @@ class LogManagerTest {
     }
     assertTrue(log.numberOfSegments > 1, "There should be more than one segment now.")
     log.updateHighWatermark(log.logEndOffset)
+    log.partitionMetadataFile.get.write(Uuid.randomUuid())
```

Comment:
Is this needed? It seems Log never reads UUID? Ditto below.

Reply:
I wanted to test that the file is actually getting deleted. In order to check it gets deleted it has to be created.

Reply:
Got it. Since the partitionMetadataFile is now created on demand, perhaps we could just change the math on the expected number of files?

Reply:
So remove the creation and subtract one file?

Reply:
That's what I am thinking. It's kind of weird to add the new file just to get the existing math works.